cfpb/consumerfinance.gov
# Add ARIA label field to Hyperlink atom

One of the things we learned in the course of the accessibility mini-project is that we can use the `aria-label` attribute on `<a>` elements to provide additional context to screen reader users in cases where the cleared link text was something like "Learn more" or when including the context would significantly harm the experience for sighted users (e.g., [repetitive links in a long table](https://www.consumerfinance.gov/policy-compliance/guidance/supervision-examinations/#anchor_manual)).

This PR adds an ARIA label field to the Hyperlink atom that's found in many modules so that Wagtail editors can add a value for the `aria-label` attribute when appropriate.

## Additions

- New ARIA label field on Hyperlinks and the template update to output it


## Testing

1. Pull branch
1. Migrate
1. Add/edit any module that includes a Hyperlink
1. Preview it without adding anything to the ARIA label field and see that it has no `aria-label` attribute
1. Enter something in the ARIA label field and preview it
1. Observe that it now has an `aria-label` attribute

## Screenshots

![screen shot 2018-05-07 at 11 59 15](https://user-images.githubusercontent.com/1044670/39711572-16b06784-51ee-11e8-9a4b-65a6e92fe8a2.png)

## Notes

- Unfortunately, this doesn't do anything for links added within a rich text field. After the move to Wagtail 2.0, we should consider a Draftail plugin that would enable that.

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
